docs/product/components/usercard.html

```diff
@@ -0,0 +1,189 @@
+---
+layout: page
+title: User Cards
+description: User cards are a combination of a user and metadata about the user or post
+---
+
+<section class="stacks-section">
+    {% header "h2", "Regular" %}
+    <div class="stacks-preview">
+{% highlight html %}
+
+    <div class="s-usercard">
```

Comment:
Slide all this whitespace over to the left. They're super indented since `pre` takes all the spacing literally.